.github/workflows/java-security.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
             ]
 
       - name: Generate settings.yaml File Inside Repository
-        uses: fertrig/create-file-action@1.0.2
+        uses: 1arp/create-a-file-action@v0.4.5
```

Comment:
@sdh100shaun @JoshuaLicense Is there any reason why we need to use fertrig's repository which is forked from 1arp's instead of the main one? 1arp's repository is updated to use node 20 version, when fertrig's is deprecated. 

https://github.com/fertrig/create-file-action
https://github.com/1arp/create-a-file-action

I cannot test 1arp's workflow, because it's not allowed in settings.  Only this version is allowed: 1arp/create-a-file-action@0.2

Thanks for answer guys :)